metaworld/policies/mujoco/sawyer_xyz/base.py

```diff
@@ -0,0 +1,67 @@
+import abc
+import warnings
+
+import numpy as np
+
+from metaworld.policies.base import Policy
+
+
+class SawyerXYZPolicy(Policy, metaclass=abc.ABCMeta):
```

Comment:
I think this superclass might actually make these policies harder to understand. ideally we want them to be easily-understandable using only the single class for each env.

Can you make any of these things helper functions?

Reply:
Yes, but leaving the `get_action` implementation up to each individual class will inevitably make them longer in terms of lines of code

Reply:
i think that is a worthy sacrifice. There is a lot of value in having, for each environment, a single flat file you can read to understand exactly what to do to solve that environment.

There are some helper functions you can define, such as extract_obs and extract_act which will eliminate 90% of this class.